docs/src/main/asciidoc/liquibase.adoc

```diff
@@ -0,0 +1,219 @@
+
+[source,properties]
+--
+quarkus.datasource.driver=org.h2.Driver
+quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:default
+quarkus.datasource.username=username-default
+quarkus.datasource.min-size=3
+quarkus.datasource.max-size=13
+
+quarkus.datasource.users.driver=org.h2.Driver
+quarkus.datasource.users.url=jdbc:h2:tcp://localhost/mem:users
+quarkus.datasource.users.username=username1
+quarkus.datasource.users.min-size=1
+quarkus.datasource.users.max-size=11
+
+quarkus.datasource.inventory.driver=org.h2.Driver
+quarkus.datasource.inventory.url=jdbc:h2:tcp://localhost/mem:inventory
+quarkus.datasource.inventory.username=username2
+quarkus.datasource.inventory.min-size=2
+quarkus.datasource.inventory.max-size=12
+
+# Liquibase configuration for the default datasource
+quarkus.liquibase.schemas=DEFAULT_TEST_SCHEMA
+quarkus.liquibase.change-log=db/changeLog.xml
+quarkus.liquibase.migrate-at-start=true
+
+# Liquibase configuration for the "users" datasource
+quarkus.liquibase.users.schemas=USERS_TEST_SCHEMA
+quarkus.liquibase.users.change-log=db/users.xml
+quarkus.liquibase.users.migrate-at-start=true
+
+# Liquibase configuration for the "inventory" datasource
+quarkus.liquibase.inventory.schemas=INVENTORY_TEST_SCHEMA
+quarkus.liquibase.inventory.change-log=db/inventory.xml
+quarkus.liquibase.inventory.migrate-at-start=true
+--
+
+Notice there's an extra bit in the key.
+The syntax is as follows: `quarkus.liquibase.[optional name.][datasource property]`.
+
+NOTE: Without configuration, Liquibase is set up for every datasource using the default settings.
+
+== Using the Liquibase object
+
+In case you are interested in using the `Liquibase` object directly, you can inject it as follows:
+
+NOTE: The Liquibase object is the instance of the org.quarkus.liquibase.Liquibase class. This kind of wrapper object to handle the
+JDBC connection for the liquibase.Liquibase object. The instance open and close the database connection for each method separately.
+
+NOTE: If you enabled the `quarkus.liquibase.migrate-at-start` property, by the time you use the Liquibase instance,
+Quarkus will already have run the migrate operation
+
+[source,java]
+--
+import org.quarkus.liquibase.Liquibase
+
+@ApplicationScoped
+public class MigrationService {
+    // You can Inject the object if you want to use it manually
+    @Inject
+    Liquibase liquibase; <1>
+
+    @Inject
+    @LiquibaseDataSource("inventory") <2>
+    Liquibase liquibaseForInventory;
+
+    @Inject
+    @Named("liquibase_users") <3>
+    Liquibase liquibaseForUsers;
+
+    public void checkMigration() {
+        // Use the liquibase instance manually
+        liquibase.dropAll(); <4>
+        liquibase.validate();
+        liquibase.update();
+        // Get the list of liquibase change set statuses
+        List<ChangeSetStatus> status = liquibase.getChangeSetStatuses(); <5>
+    }
+}
+--
+
+<1> Inject the Liquibase object if you want to use it directly
```

Comment:
Better move the deep technical explanation here

Reply:
Done